src/tests/system/tests/test_netgroups.py

```diff
@@ -108,3 +124,282 @@ def test_netgroups__add_remove_netgroup_member(client: Client, provider: Generic
     assert len(result.members) == 1
     assert "(-, user-1)" not in result.members
     assert "(-, user-2)" in result.members
+
+
+@pytest.mark.importance("low")
+@pytest.mark.topology(KnownTopology.LDAP)
+def test_netgroup__netgroup_nisnetgrouptriple(client: Client, ldap: LDAP):
+    """
+    :title: Netgroup with nisNetgroupTriple
+    :setup:
+        1. Create users, groups and start sssd.
+    :steps:
+        1. Check nisNetgroupTriple contains members as added in the test.
+    :expectedresults:
+        1. NisNetgroupTriple should contain members as added in the test.
+    :customerscenario: False
+    """
+    ou = ldap.ou("Netgroup")
+    create_users(ldap)
+
+    qa_users = ldap.netgroup("QAUsers", basedn=ou).add()
+    qa_users.add_member(host="testhost1", user="ng9000", domain="ldap.test")
+
+    client.sssd.start()
+
+    assert "(testhost1, ng9000, ldap.test)" in client.tools.getent.netgroup("QAUsers").members
+
+
+@pytest.mark.importance("low")
+@pytest.mark.topology(KnownTopology.LDAP)
+def test_netgroup__cache_time_out(client: Client, ldap: LDAP):
+    """
+    :title: Decrease the cache time out and add new entry for nisNetgroupTriple
+    :setup:
+        1. Create users, groups and start sssd.
+    :steps:
+        1. Check if the netgroup "QAUsers" contains the tuple (testhost1, ng9000, ldap.test),
+            verifying that the member added earlier exists.
+        2. Check that (testhost1, ng9001, ldap.test) does not exist in the netgroup,
+            confirming that the user ng9001 has not yet been added.
+        3. Add another member to the "QAUsers" netgroup.
+        4. Check if after the cache expires, the user ng9001 and host testhost2
+            are now correctly considered part of the netgroup "QAUsers".
+        5. Check if both tuples (testhost1, ng9000, ldap.test) and (testhost2, ng9001, ldap.test)
+            are present in the "QAUsers" netgroup after the changes.
+    :expectedresults:
+        1. Netgroup "QAUsers" contains the tuple (testhost1, ng9000, ldap.test)
+        2. Tuple (testhost1, ng9001, ldap.test) does not exist in the netgroup
+        3. Another member should be added to the "QAUsers" netgroup.
+        4. The user ng9001 and host testhost2 are now correctly considered part of the netgroup "QAUsers".
+        5. Both tuples (testhost1, ng9000, ldap.test) and (testhost2, ng9001, ldap.test)
+            are present in the "QAUsers" netgroup after the changes.
+    :customerscenario: False
+    """
+    ou = ldap.ou("Netgroup")
+    create_users(ldap)
+
+    qa_users = ldap.netgroup("QAUsers", basedn=ou).add()
+    qa_users.add_member(host="testhost1", user="ng9000", domain="ldap.test")
+
+    client.sssd.dom("test")["entry_cache_timeout"] = "60"
```

Comment:
While it is more realistic to set it to 60, we can reduce the time that it takes to run this test and set it to 4 reducing the time for it to run this test by at least 56 seconds. 

This test is already covered by the background refresh tests. 
https://github.com/SSSD/sssd/blob/b928dbe1fcb6cf4b7acbd97d3df9514d5c554953/src/tests/system/tests/test_cache.py#L27

We do not need this test.